backend/src/main/java/com/example/backend/controller/GifController.java

```diff
@@ -19,8 +18,17 @@ public GifController(GifService gifService) {
     }
 
     @GetMapping
-    public List<Gif> getListOfGifs(){return this.gifService.getListOfGifs();
+    public List<Gif> getListOfGifs() {
+        return this.gifService.getListOfGifs();
     }
 
+    @PostMapping
+    public List<Gif> addNewParty(@RequestBody GifWithoutId gifWithoutId){
+        this.gifService.addGif(gifWithoutId);
+        return this.gifService.getListOfGifs();
```

Comment:
I think it's best practice to return the new added object in the add method instead of returning the whole list; of course if you don't need the list in your code...